backend/tests/test.cpp

```diff
@@ -0,0 +1,29 @@
+#define CATCH_CONFIG_MAIN
+#include <regex>
+#include <catch2/catch_all.hpp>
+#include "../src/passwords.cpp"
+
+std::unordered_set<std::string> passwordSet = passwords::generatePasswords(5, 24);
+
+bool hasLettersAndDigit(std::string str)
```

Comment:
**suggestion**: This string is const reference. Use a more descriptive parameter name, `password`.